fdbclient/ClientLogEvents.h

```diff
@@ -67,6 +75,38 @@ namespace FdbClientLogEvents {
 		}
 	};
 
+	struct EventGetVersion_V2 : public Event {
```

Comment:
Could we either change the name of this or add a comment to indicate that this is the version as of 6.2?

Reply:
I will add a comment